app/controllers/hub/efile_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Hub
+  class EfileController < Hub::BaseController
+    include FilesConcern
+    authorize_resource
+    load_resource except: [:index, :show]
```

Comment:
but actually for lines 4 and 5 we can just replace it with a single `load_and_authorize` call and we will have to specify the name of the resource/model we are using since it doesn't match the name of controller "Efile" so I would recommend replacing these two lines with: 

```
load_and_authorize_resource :efile_submission, through: :intake, only: [:index, :show]
```
might have to play around with the through intake part to make sure its correct, it would be good to add a test case if not now then eventually that will check that a user can access the correct selection of efile submissions on this page

Reply:
Done